scene/2d/line_builder.cpp

```diff
@@ -108,13 +62,14 @@ void LineBuilder::build() {
 	const float hw_sq = hw * hw;
 	const float sharp_limit_sq = sharp_limit * sharp_limit;
 	const int len = points.size();
+	bool wrap_around = closed && len > 2;
```

Comment:
```suggestion
	const bool wrap_around = closed && len > 2;
```
Useless, but consistent with the above.

Reply:
afaik we don't do const for primitives (bool, int, float). I'll remove it above instead I think?

Reply:
We don't use it for arguments, but for local variables is fine.
Either way it doesn't matter, but keeping consistency here requires less changes.